### DISCUSSION TITLE
Welcome to dosbian Discussions!

### DISCUSSION THREAD

**Discussion by weese** [weese/dosbian#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by actraiser30** [comment]
Hi dosbian image missing all dosbox except dosboxX,also cannot update the current build the internet lan connection is disabled in the image,and also cannot build/compile a newest image as I don't have a docker and the building fails of course,can you upload the updated image to downloads please?
